src/pages/api/properties.js

```diff
@@ -0,0 +1,12 @@
+import { getProperties } from '../../utils/service-api-client/repairs'
+
+export default async function endpoint(req, res) {
```

Comment:
Just fyi I think latest syntax would be `export default async (req, res) => {}` but not a big deal 😄

Reply:
Thanks. [Updated](https://github.com/LBHackney-IT/repairs-hub-frontend/pull/26/commits/35b2262c25a2b8c62917330d05a31eeec5f9fcfa#diff-f60e89599c96f95b247f9e10b8008733afccbe32faff1ee0ad7ffb502c67ae39R3)